Geometry/MTDNumberingBuilder/test/MTDTopologyAnalyzer.cc

```diff
@@ -90,20 +92,29 @@ void MTDTopologyAnalyzer::analyze(const edm::Event& iEvent, const edm::EventSetu
 
     size_t num = fv.geoHistory().size();
 
-    if (fv.geoHistory()[num - 1].logicalPart().name() == "BarrelTimingLayer") {
+    if (fv.geoHistory()[num - 1].logicalPart().name().name() == "BarrelTimingLayer") {
```

Comment:
Is there a reason this interface requires calling `name()` twice? It seems suboptimal.

Reply:
@kpedro88 this is the old DDD syntax to strip the namespace. Its absence was causing the bulk of the test to be ineffective. In DD4hep this is handled in a different way (@ianna can better comment in case)

Reply:
@kpedro88 and @fabiocos - yes, a ```DDLogicalPart``` has a ```name()``` which is a ```DDName```. The latter is a two strings object: a ```namespace()``` and a ```name()```. Actually, every DD object has a ```DDName```.